functions/venue.js

```diff
@@ -113,6 +115,7 @@ const createVenueData = (data, context) => {
     profile_questions: data.profile_questions,
     placement: { ...data.placement, state: PlacementState.SelfPlaced },
     showLiveSchedule: data.showLiveSchedule ? data.showLiveSchedule : false,
+    type: data.type === UNCLICKABLE_ROOM_TYPE,
```

Comment:
What is the intent of this change here? I don't think it is correct.. but it would be good to understand what you were meaning to do with it/why.

Reply:
From reading the rest of the code in this PR I can now understand that you chose to add a new boolean field to `Venue`, which isn't the solution here. `type` exists on `Room`, and is an enum of `RoomTypes`

```suggestion
```